### DISCUSSION TITLE
OnEdgeClick for edges with in a group

### DISCUSSION THREAD

**Discussion by vipivedanta** [xyflow/xyflow#2435] [General]
For edges outside a 'group' node , I am getting the prop triggered, but for an edge between two nodes inside a parent 'group' node, I can't even click the edge, instead I get the onElementClick triggered. 

Is there any other prop specifically for the edges inside a group ?

**Comment by moklick** [comment]
By default edges are behind groups. You can set a `zIndex` option for edges to display them on top of nodes.

**Reply by vipivedanta** [reply]
Yes, thank you!

**Comment by nanohealth-7844** [comment]
> By default edges are behind groups. You can set a `zIndex` option for edges to display them on top of nodes.

Is there any method to do edge clickable ?

**Reply by moklick** [reply]
an edge is clickable. What's the question?

**Reply by nanohealth-7844** [reply]
i want to open a pop screen on click on the edge of connecting two nodes 
so , i want to make edge is clickable how can i do this

**Reply by bcakmakoglu** [reply]
Edges are already clickable. Maybe you're looking for the `onEdgeClick` prop? 🤔

**Reply by nanohealth-7844** [reply]
Can i get your linkedIn profile link for connection

**Reply by moklick** [reply]
🤨
